packages/opentelemetry-exporter-collector/src/transformMetrics.ts

```diff
@@ -154,15 +154,14 @@ export function toHistogramPoint(
   metric: MetricRecord,
   startTime: number
 ): opentelemetryProto.metrics.v1.HistogramDataPoint {
-  const histValue = metric.aggregator.toPoint().value as Histogram;
+  const histPoint = metric.aggregator.toPoint();
```

Comment:
```typescript
const { value, timestamp } = metric.aggregator.toPoint();
```

Reply:
Is there any way to cast `value` in-line to be of type Histogram? That way, I don't have to do `(value as Histogram).sum`, `(value as Histogram).min`, etc.

Reply:
```typescript
const { value, timestamp } = metric.aggregator.toPoint() as { value: Histogram, timestamp: number };
```

Reply:
Thanks!